modules/host/negotiatesettings.go

```diff
@@ -26,54 +26,60 @@ func (h *Host) capacity() (total, remaining uint64, err error) {
 	return total, remaining, nil
 }
 
+// externalSettings compiles and returns the external settings for the host.
+func (h *Host) externalSettings() (modules.HostExternalSettings, error) {
+	totalStorage, remainingStorage, err := h.capacity()
+	if err != nil {
+		return modules.HostExternalSettings{}, err
+	}
```

Comment:
uh, yeah. I didn't think too much of it, but I might be able to drop that error.